docs/en/deploy/Running-Alluxio-On-Yarn.md

```diff
@@ -6,14 +6,16 @@ group: Deploying Alluxio
 priority: 6
 ---
 
-TODO: Expand the explanation of why we prefer to run alongside YARN instead of on top. Relevant
-mailing list thread: https://groups.google.com/forum/#!topic/alluxio-users/KEvBE9AiJlE
-TODO: Tell users how to configure Alluxio servers running on Yarn
-TODO: Discuss how to configure the YARN application classpath so that applications can work with Alluxio.
-TODO: Add troubleshooting help
-- How to debug when alluxio-yarn.sh fails to bring up Alluxio servers
-- How to find logs when Alluxio containers fail to start
-- How to diagnose when there aren't enough YARN resources to launch Alluxio containers
+This document discusses how to run Alluxio in a YARN environment. There are two
+approaches: running Alluxio standalone, and running Alluxio as an application
+on top of YARN. We support both approaches, but recommend running Alluxio
+standalone for a few reasons:
+
+- Alluxio's architecture is designed to have one worker per host, but this is not
+supported by YARN.
+- Running Alluxio on top of YARN adds the additional complexity of managing Alluxio's
```

Comment:
how about fault tolerance? I remember we don't support restart workers with Alluxio on YARN

Reply:
added